--- a/src/ruby/tools/bin/grpc_tools_ruby_protoc
+++ b/src/ruby/tools/bin/grpc_tools_ruby_protoc
@@ -23,8 +23,7 @@
 
 plugin_name = 'grpc_ruby_plugin' + ext
 
-protoc_dir = File.join(File.dirname(__FILE__),
-                       PLATFORM.architecture + '-' + PLATFORM.os_name)
+protoc_dir = '/usr/bin'
 
 protoc_path = File.join(protoc_dir, protoc_name)
 
--- a/src/ruby/end2end/package_with_underscore_checker.rb
+++ b/src/ruby/end2end/package_with_underscore_checker.rb
@@ -21,10 +21,10 @@ def main
 
   fail 'CONFIG env variable unexpectedly unset' unless ENV['CONFIG']
   bins_sub_dir = ENV['CONFIG']
-  bins_dir = File.join(root_dir, 'bins', bins_sub_dir)
+  bins_dir = '/usr/bin'
 
   plugin = File.join(bins_dir, 'grpc_ruby_plugin')
-  protoc = File.join(bins_dir, 'protobuf', 'protoc')
+  protoc = File.join(bins_dir, 'protoc')
 
   got = nil
 
